fabulaws/library/wsgiautoscale/servers.py

```diff
@@ -292,20 +288,31 @@ def install_less_and_yuglify(self):
         and the ``yuglify`` binary used by pipeline.
         """
 
-        self.add_ppa('ppa:chris-lea/node.js')
-        #In 12.04 npm is included in the chris-lea ppa in the 'nodejs' package
-        self.install_packages(['nodejs'])
+        if self.ubuntu_release >= Decimal('16.04'):
+            # nodejs-legacy contains 'node' -> 'nodejs' symlink on 16.04
+            self.install_packages(['npm', 'nodejs-legacy'])
+        else:
+            self.add_ppa('ppa:chris-lea/node.js')
```

Comment:
Unrelated to this PR, but the chris-lea PPA is deprecated and NodeSource hosts the updated one: https://github.com/nodesource/distributions